src/api/spec/controllers/webui/cloud/azure/configurations_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+# WARNING: Some tests require real backend answers, so make sure you uncomment
+# this line and start a test backend.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Webui::Cloud::Azure::ConfigurationsController, type: :controller, vcr: true do
+  let!(:user) { create(:confirmed_user, login: 'tom') }
+
+  before do
+    login(user)
+  end
+
+  describe 'GET #show' do
+    context 'without Azure configuration' do
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          get :show
+        end
+      end
+
+      it 'creates an Azure configuration' do
+        expect(assigns(:azure_configuration)).not_to be_nil
+      end
+    end
+
+    context 'with Azure configuration' do
+      let!(:azure_configuration) { create(:azure_configuration, user: user) }
+
+      before do
+        Feature.run_with_activated(:cloud_upload, :cloud_upload_azure) do
+          get :show
+        end
+      end
+
+      after do
```

Comment:
Why do you do this? The database will be reset after each test.

Reply:
It gets cleaned up as soon as rspec has finished, but we perform randomly (rspec seeds) chosen actions (`#destroy`, `#put`, `#get`) on that object and therefor, the tests will sometimes fail if you don't set this up every time from the scratch. We heavily debugged this and that was the only solution we've found. ( @eduardoj )

Reply:
> It gets cleaned up as soon as rspec has finished

Not according to [spec/support/database_cleaner.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/support/database_cleaner.rb#L24)

Reply:
@hennevogel 

> Why do you do this? The database will be reset after each test.

I checked again, and this time there was no problems when removing the `after` clause. Removed.